### ISSUE TITLE
lib/theme-and-appearance.zsh sets non-appearance options

### ISSUE THREAD

**Issue by apjanke** [ohmyzsh/ohmyzsh#3586]
The `lib/theme-and-appearance.zsh` script sets some shell options that don't look related to cosmetic theme and appearance stuff. At line 22:

```
#setopt no_beep
setopt auto_cd
setopt multios
setopt cdablevars
```

Shouldn't these be set somewhere else? They have significant semantic effects, especially `multios`.

**Comment by 7rulnik** [comment]
I believe that `auto_cd` should be inside `lib/directories.zsh`

**Comment by mcornella** [comment]
Fixed in 277f38212aef31a6baba2cf1a0a355af611be5e0
